### ISSUE TITLE
SMTP Timeout

### ISSUE THREAD

**Issue by MrVampCruz** [mfbx9da4/deep-email-validator#28]
Running my server locally is fine, but after running my express server online, smtp timeout, and it send validity as false

**Comment by julienfouilhe** [comment]
I had the same problems when I wanted to find out why our emails bounced on, and I found out that the SMTP port is blocked by a lot of IsP, apparently exactly so you can't do this and bruteforce the email addresses out of a domain for instance.

IMO this info should be in this dependency's README.

**Comment by bulog** [comment]
Yes, mostly the ISP block the connections between your server to the SMTP server by the port 25(default). You should ask them to remove the block

**Comment by acoyfellow** [comment]
It'd be a great feature if we could customize SMTP port. For Google Cloud Platform, we're allowed to use ports 587 or 465. 

- https://cloud.google.com/compute/docs/tutorials/sending-mail

**Comment by msadovyi** [comment]
Agree! Please, add smtp port to the options.

**Comment by ymath-xyz** [comment]
any new solution?

**Comment by fabioselau077** [comment]
> any new solution?

has?

**Comment by sharukhkhanajm** [comment]
any update on this?

**Comment by abdullahalydev** [comment]
I tried to use mobile data instead of my home's internet and it worked
![image](https://github.com/user-attachments/assets/8e5ee5b5-4622-4d06-84fd-055d2cca4a4c)

**Comment by mfbx9da4** [comment]
Sorry for the late reply. Most cloud hosting providers (Heroku, AWS, GCP, Azure, etc.) block outbound port 25 to prevent spam. This causes SMTP validation to always time out when deployed, even if it works locally.

Workaround:
```js
await validate({ email: '...', validateSMTP: false })
```

This isn't something the package can control — it's a network-level restriction on your hosting provider's side.
